ubiquity/dell-bootstrap.py

```diff
@@ -797,6 +850,21 @@ def ok_handler(self):
 
         return Plugin.ok_handler(self)
 
+    def dhc_ok_handler(self):
```

Comment:
As I can tell you're not actually using this function.  Is that a mistake?

Reply:
Yes, don't need this function, I add it for doing  unit test in developer progress,  and now should delete it, thanks.